src/app/pages/product-item.page/product-item.page.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-product-item.page',
```

Comment:
NIT: I would replace the dot (.) with a dash (-). Component names are normally using snake-case format.